app/models/config.rb

```diff
@@ -58,7 +48,7 @@ def help_text
   end
 
   def self.autosetup
-    CONFIG_FIELDS.map(&:to_s).each do |field|
+    config_keys.keys.each do |field|
```

Comment:
Nit: I'd consider doing `each_key` instead of `keys.each` here.

Reply:
Less nit: we should consider renaming "config_keys" to "config_fields" as that's less implementation detail and more "what is this thing to our users"

Reply:
I'm _really_ reluctant to do field renaming in the scope of this PR given the parts in motion - renaming fields would mean we'd have to crack open controllers to get this to work and I want to keep those as stable as possible. It feels like the pain of renaming outweighs the benefit by quite a bit here but maybe I'm missing something @mdworken ?

Reply:
I second the reluctance - let's file an issue on our backlog to do after we're six months stable on postgres, not try to be too cute now